--- hyprland.conf	2025-07-20 20:06:29.776206897 +0800
+++ hyprland_l.conf	2025-07-20 20:19:25.378722640 +0800
@@ -36,14 +36,14 @@
     inactive_opacity = 1.0
 
     shadow {
-        enabled = true
+        enabled = false
         range = 4
         render_power = 3
         color = rgba(1a1a1aee)
     }
 
     blur {
-        enabled = true
+        enabled = false
         size = 3
         passes = 1
 
@@ -53,7 +53,7 @@
 }
 
 animations {
-    enabled = true
+    enabled = false
 
     bezier = myBezier, 0.05, 0.9, 0.1, 1.05
 
@@ -75,7 +75,7 @@
 }
 
 misc {
-    vfr = true
+    vfr = false
     force_default_wallpaper = -1
     disable_hyprland_logo = false
 }
@@ -157,5 +157,3 @@
 bind =, xf86audiomute, exec, wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle
 
 windowrulev2 = suppressevent maximize, class:.*

-layerrule {
-    name = Blur layers
-    match:namespace = (launcher|waybar)
-    blur = on
-}
